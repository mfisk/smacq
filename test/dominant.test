Pairs   :- pcapfile(0325@1112-snort.pcap) | uniq(srcip dstip)
SrcCount :- count() group by ipprotocol srcport
DstCount :- count() group by ipprotocol dstport
Pdf     :- filter(count) | pdf
Print   :- sort(-r probability) | print(type ipprotocol port probability)

Pairs | SrcCount | const(-f type src) | Pdf | rename (srcport port) | Print
Pairs | private | DstCount | const(-f type dst) | Pdf | rename (dstport port) | Print
