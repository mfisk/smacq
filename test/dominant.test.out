src	tcp	1043	0.333333
src	tcp	1051	0.333333
src	udp	137	0.333333
dst	tcp	139	0.333333
dst	tcp	445	0.333333
dst	udp	137	0.333333
